Scripts/script-ExtractIndicatorsFromWordFile.yml

```diff
@@ -0,0 +1,121 @@
+              self.file_name = file_res['Contents']['name']
+
+      def convert_doc_to_docx(self):
+          conversion_res = subprocess.check_output(['soffice', '--headless', '--convert-to', 'docx', self.file_path])
+          output_file_name = self.file_name[0:self.file_name.rfind('.')] + '.docx'
+          self.file_path = self.file_path + ".docx"
+          with open(self.file_path, 'rb') as f:
+              contents = f.read()
+              f_data = f.read()
+              self.res = fileResult(output_file_name, f_data)
+
+      def extract_indicators(self):
+          try:
+              document = Document(self.file_path)
+              self.all_data = self.get_paragraphs(document)
+              self.all_data += self.get_tables(document)
+              self.all_data += self.get_core_properties(document)
+
+              output_file_name = self.file_name[0:self.file_name.rfind('.')] + '.txt'
+              self.res = fileResult(output_file_name, self.all_data.encode('utf8'))
+          except PackageNotFoundError as e:
+              self.errEntry["Contents"] = "Input file is not a valid docx/doc file."
+              self.res = self.errEntry
+          except BaseException as e:
+              self.errEntry["Contents"] = "Error occurred while parsing input file.\nException info: " + str(e)
+              self.res = self.errEntry
+
+      def get_paragraphs(self, document):
+          return '\n'.join([para.text for para in document.paragraphs])
+
+      def get_tables(self, document):
+          all_cells_txt = ""
+          if document.tables:
+              for table in document.tables:
+                  for row in table.rows:
+                      for cell in row.cells:
+                          for para in cell.paragraphs:
+                              all_cells_txt += (" " + para.text)
+          return " ".join(all_cells_txt.split())  # Removes extra whitespaces
+
+      def get_core_properties(self, document):
+          all_properties_txt = document.core_properties.author + " " +\
+          document.core_properties.category + " " +\
+          document.core_properties.comments + " " +\
+          document.core_properties.identifier + " " +\
+          document.core_properties.keywords + " " +\
+          document.core_properties.subject + " " +\
+          document.core_properties.title + " "
+          return " ".join(all_properties_txt.split())
+
+      def parse_word(self):
+          self.get_file_details()
+          if self.file_name.endswith(".doc"):
+              self.convert_doc_to_docx()
+              self.extract_indicators()
+          elif self.file_name.endswith(".docx"):
+              self.extract_indicators()
+          else:
+              return_error("Input file is not a doc file.")
+
+  parser = WordParser()
+  parser.parse_word()
+  # Extract indicators (omitting context output, letting auto-extract work)
+  demisto.results(str(demisto.executeCommand("extractIndicators", {
```

Comment:
Return as human readable

Reply:
Return all data
Indicators
errors
Parsed data

Reply:
OK, but why both the indicators and the full data?